.vscode/settings.json

```diff
@@ -12,5 +12,10 @@
     "joypad",
     "input",
     "forms"
-  ]
+  ],
+  "Lua.runtime.version": "LuaJIT",
+  "Lua.workspace.library": [
+    "${3rd}/love2d/library"
+  ],
+  "Lua.workspace.checkThirdParty": false
```

Comment:
These three settings are for the sumneko.lua extension, which you used to have in the recommendations but changed to trixnz.vscode-lua without removing these extension settings. Assuming the person *does* install sumneko's Lua extension there's still some issues with these settings as follows:
- After doing a couple of checks, i've determined that bizhawk runs lua5.1, so the runtime version setting here is incorrect
- Not sure exactly what "Lua.workspace.library" does to be honest, but from what i gathered from a quick search it allows the extension to load external libraries. Not sure what the library you specified here is but i'm pretty sure it's not being used in the tracker code. Not sure if this library would be an extra thing we'd need to install or not / not sure about if this setting is at all needed to be honest

The "Lua.workspace.checkThirdParty" setting is fine (if using sumneko's Lua extension) since as you mentioned this blocks the little "Do you need to configure your work environment as 'LÖVE'?" popup from vscode

Reply:
thanks for the review @Fellshadow. I've updated the lua version specified, and i've also removed the workspace library stuff (it was added by default when the popup showed and i clicked apply to workspace.

Reply:
ah ok that makes sense then, still useful either way i suppose.
I'm happy with these changes now, though gonna merge to staging instead of main since our workflow now is to push to bugfixes/staging depending on the type of changes and then ultimately pushing things from staging (which regularly gets updated from bugfixes) -> main when gearing up for a new release (which there'll be one soon). Not that it makes a whole lot of difference for these changes, just more for consistency